classes/classes/Player.as

```diff
@@ -770,7 +770,7 @@ use namespace kGAMECLASS;
 			{
 				race = "lizan";
 			}
-			if (dragonScore() >= 6)
+			if (dragonScore() >= 7)
```

Comment:
Forgot to check a checkbox again after some local tests. Hence the 2 commits here.
Anyway: Tested and working without issues now.